README.md

```diff
@@ -1,6 +1,6 @@
 # HSTCAL
 
-[![Jenkins CI](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/master/badge/icon)](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/master/)
+[![Jenkins CI](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/main/badge/icon)](https://ssbjenkins.stsci.edu/job/STScI/job/hstcal/job/main/)
```

Comment:
Is ssbjenkins still a thing?

Reply:
good catch; I'm not sure there is any badge that we can serve from behind the network, at the moment:
```suggestion
```

Reply:
I thought one of the Chief Engineers killed it altogether.